### ISSUE TITLE
Right-click over image has a weird effect

### ISSUE THREAD

**Issue by fredck** [ckeditor/ckeditor5-image#30]
With Chrome, when right-clicking an image ([like the one in the demo](https://ckeditor5.github.io/)), the native selection seems to briefly take place and the fake selection disappears. It gives a "flashing" effect. Then the native context menu appears, which is ok.

Ideally, the fake selection would stay on the image and no fancy flashing should happen.

**Comment by Reinmar** [comment]
Hmm... this, I'm afraid, will be very hard to do. We'd need to figure out that context menu was opened and then somehow block some processes in the editor. The solution may require a lot of hacks to make it work on all browsers and I'm pretty sure it will be unstable.

Anyway, it's a small issue, so fortunately we're not forced to work on it.

**Comment by fredck** [comment]
True. Maybe we can at least avoid the native selection to happen... the whole image covered by a transparent blue layer. It happens for a few milliseconds - almost unnoticeable, but it gives the flashing effect.

Then, the fact that it loses our fake selection border is not a big deal for now.

**Comment by Reinmar** [comment]
> Maybe we can at least avoid the native selection to happen...

I didn't understand what you mean first, because I haven't even noticed that it blinks. But that's true – we could try to fix this part.

**Comment by jodator** [comment]
@fredck Could you verify the issue is still visible? I might not see this now and we have done some change to the way how selection works.

**Comment by fredck** [comment]
> Could you verify the issue is still visible?

Yes, I still see this in the [current online demo](https://ckeditor5.github.io/) (v10.1.0).
